pandas/core/internals/blocks.py

```diff
@@ -428,7 +428,7 @@ def split_and_operate(self, func, *args, **kwargs) -> list[Block]:
     # Up/Down-casting
 
     @final
-    def coerce_to_target_dtype(self, other, warn_on_upcast: bool = False) -> Block:
+    def coerce_to_target_dtype(self, other, raise_on_upcast: bool = False) -> Block:
```

Comment:
Are there any usages of this method, outside this file, that does not set this to `raise_on_upcast=False`?

Reply:
there's no usage of this method at all outside this file

i've removed the default `=False` anyway, better to be explicit about this one